### ISSUE TITLE
One more will cost in core

### ISSUE THREAD

**Issue by millerj-tc** [ACG8/Lucid#283]
Needs a 4 drop so that players playing first core game feel like will has enough payoffs.

**Comment by millerj-tc** [comment]
Abram asked for a will gain lucid in core as well as one more will sink.

This also gives the core a high will payout goal that is still not too swingy. Or at least that's the idea.

This  would replace Ghost of the Galaxies.

![image](https://user-images.githubusercontent.com/3642851/45303060-87dbcf80-b4da-11e8-909a-b2d768c7fdb3.png)

**Comment by ANRTheBigBoy** [comment]
You really want 3 will abilities on 1 card?

**Comment by millerj-tc** [comment]
I'm not sure how to interpret your comment. Are you concerned that it's too good? That it's visually cluttered? That it's hard to parse? Too linear? That the first ability should be free?

It's just a draft of an idea so I'm not super attached to any aspect of it, I was just trying to fill some of the core needs you mentioned.

**Comment by ANRTheBigBoy** [comment]
ya I think having 3 will costs on a core-set card is a little overwhelming. The card is cool, but it feels like a draft card.

**Comment by millerj-tc** [comment]
So the only thing I don't like about this design is that it may not *feel* that good even though it's super fair and useful.

![image](https://user-images.githubusercontent.com/3642851/45363910-60e6d180-b59e-11e8-9138-f76942dd51c2.png)

**Comment by ANRTheBigBoy** [comment]
I think you can simplify it even more by removing the cost from the passive. The return on the 2nd ability is fair enough that I think that's fine.

**Comment by ACG8** [comment]
The cost on the passive is redundant. It triggers at the end of the round, so you will never have anything to spend that will on anyway.

**Comment by millerj-tc** [comment]
@abram I'm not concerned that passive -> burst is too good, I'm worried about locking this in in Cube/Whale/etc.

@ananda if I have exactly X will and spend X, then I can't use the ability if it has a cost. It is not redundant, it's just a very conditional speed bump. Basically it means that for as long as you want to gain will, you have one less will.

**Comment by ACG8** [comment]
Ok, sure. I had been thinking about it as equivalent to gaining tapped will, but it is true that it places a limit on your will usage.

I think it is fine if they want to lock it in as their economy. Missing out on burst effects is a steep price to pay for that, especially since it gives no points passively.

**Comment by millerj-tc** [comment]
Sorry let me clarify, I'm partially worried it's too good but more worried
that it's boring.

Even if it's fair, if every big will deck wants to play this and it's
optimal to lock it in until they're at 9+ will, it means that how to use it
isn't ever interesting.

**Comment by ANRTheBigBoy** [comment]
I don't know that 1 passive will is overpowered in the current build. Bursts of will are toned down aggressively because getting a resource faster has value. The fact that you get the will over time lowers its value. 
Whale/Cube decks would rather have Doomday/Chaos. This is a nice compromise. 
I think we have be operating under some assumptions around how valuable Will is that are worth putting to the test. Having the 1-will drip in the core and just seeing how that affects game balance will at the very least give us a lot of information.
